docs/en/top-erc20s.md

```diff
@@ -0,0 +1,112 @@
+---
+id: top-erc20
+title: Top ERC20s Deployed to Loom
+sidebar_label: Top ERC20s Deployed to Loom
+---
+
+Here's the list of the top 100 ERC20 smart contracts already deployed to Loom. Please scroll to the right to see all columns.
+
+| Token Name  | Decimals | MainNet | asia1 | stage | rinkeby-asia1 | rinkeby-us1 | plasma |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+Paxos Standard | 18 | 0x8e870d67f660d95d5be530380d0ec0bd388289e1 | 0x65250dc7016addd1ce226259e2e60ca8ac13ca52 | 0x56c80de2eafe0086d690b87082472548f7bb917f | 0x9d633be2ff4151e85fc5d6444b5e12f5ae442f23 | 0x08eff0e92291b70cdb5c1cfe81352aca669a9e19 | 0xd443c9cee967b5d068dbf7471c33a2e0bd135f97 |
```

Comment:
ONLY plasma chain, please remove all other networks in the markdown

Reply:
> ONLY plasma chain, please remove all other networks in the markdown

@mattkanwisher the list is updated :)